run.sh

```diff
@@ -24,16 +24,15 @@ if [ ${#ES_NODE_SIGNER_PRIVATE_KEY} -ne 64 ]; then
 fi
 
 # download blob_poseidon.zkey if not yet
-zkey_file="./ethstorage/prover/snarkjs/blob_poseidon.zkey"
+zkey_file="./build/bin/snarkjs/blob_poseidon.zkey"
 if [ ! -e  ${zkey_file} ]; then
-  echo "${zkey_file} not found. Start downloading..."
-  file_id="1ZLfhYeCXMnbk6wUiBADRAn1mZ8MI_zg-"
-  html=`curl -c ./cookie -s -L "https://drive.google.com/uc?export=download&id=${file_id}"`
-  curl -Lb ./cookie "https://drive.google.com/uc?export=download&`echo ${html}|grep -Po '(confirm=[a-zA-Z0-9\-_]+)'`&id=${file_id}" -o ${zkey_file}
-  echo "downloaded ${zkey_file}"
+echo "${zkey_file} not found, start downloading..."
+file_id="1ZLfhYeCXMnbk6wUiBADRAn1mZ8MI_zg-"
+html=`curl -c ./cookie -s -L "https://drive.google.com/uc?export=download&id=${file_id}"`
+curl -Lb ./cookie "https://drive.google.com/uc?export=download&`echo ${html}|grep -Po '(confirm=[a-zA-Z0-9\-_]+)'`&id=${file_id}" -o ${zkey_file}
```

Comment:
Need to update the option -P for macOS

Reply:
Tracked by https://github.com/ethstorage/es-node/issues/135